application/jira/v1/config/definition.json

```diff
@@ -0,0 +1,24 @@
+{
+  "availableTasks": [
+    "TASK_LIST_BOARDS",
+    "TASK_LIST_ISSUES",
+    "TASK_LIST_SPRINTS",
+    "TASK_GET_ISSUE",
+    "TASK_GET_SPRINT",
+    "TASK_CREATE_ISSUE",
+    "TASK_UPDATE_ISSUE",
+    "TASK_MOVE_ISSUE_TO_EPIC"
```

Comment:
I think "TASK_MOVE_ISSUE_TO_EPIC" can be merged in TASK_UPDATE_ISSUE conceptually.